pyomo/solvers/tests/testcases.py

```diff
@@ -120,7 +120,7 @@
      "(in 2.10.x).")
 
 ExpectedFailures['cbc', 'nl', 'LP_unbounded'] = \
-    (lambda v: v[:2] == (2, 10),
+    (lambda v: v > (2, 10) and v < (2, 10, 6),
      "Cbc fails (invalid free()) for unbounded LP models through "
      "the NL interface in 2.10.x versions "
```

Comment:
Can you update this comment with the range of CBC versions this test is skipped for?

Reply:
Done.